src/index.js

```diff
@@ -10,5 +10,12 @@ import "./stylesheets/main.scss";
 
 import {MaterialsDesignerContainer} from './MaterialsDesignerContainer';
 
-// Store component reference in window to access it in console for debugging/tests purposes
-window.MDContainer = ReactDOM.render(<MaterialsDesignerContainer/>, document.getElementById('root'));
+/*
+ * Set timeout to ensure Codemirror CSS is loaded: https://github.com/graphql/graphiql/issues/33#issuecomment-318188555
+ * CSS is loaded in the component using CodeMirror (eg. `Basis`).
+ */
+setTimeout(() => {
```

Comment:
Is CM CSS gonna be an issue when using compiled MD in webapp? Let's do `npm run transpile` and then rsync `dist` dir with webapp `node_modules/@exabyte-io/materials-designer/dist` dir and restart webapp to make sure it works fine.